### ISSUE TITLE
After sorting by album artist, sort by year

### ISSUE THREAD

**Issue by neubland** [digimezzo/dopamine-windows#1061]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report  
[X] Feature request
</code></pre>

## I'm using these versions

* Windows version: 10 1903
* Dopamine version: 2 Preview 15

## Reproduction steps

* Sort albums by album artist and open the album view

## Observed behavior
Albums from the same album artist seem to be randomly ordered

## Desired behavior
Sort albums that are grouped together as they are from the same album artist by date (date would be better than year because some artists release multiple albums in one year)

**Comment by chadfranklin47** [comment]
You assign dates to your music tags?

**Comment by neubland** [comment]
Well yeah, release date, or at least release year. Some programs like Musicbrainz Picard can help automating this process. Sorting anything chronologically is not that strange.

**Comment by chadfranklin47** [comment]
Yeah, by year for sure. I only ever see dates in albums I buy from itunes though.
